src/coreclr/nativeaot/Runtime/DebugHeader.cpp

```diff
@@ -269,7 +269,8 @@ extern "C" void PopulateDebugHeaders()
 
     static_assert(MethodTable::Flags::EETypeKindMask         == 0x00030000, "The debugging data contract has a hard coded dependency on this value of MethodTable::Flags. If you change this value you must bump major_version_number.");
     static_assert(MethodTable::Flags::HasFinalizerFlag       == 0x00100000, "The debugging data contract has a hard coded dependency on this value of MethodTable::Flags. If you change this value you must bump major_version_number.");
-    static_assert(MethodTable::Flags::HasPointersFlag        == 0x00200000, "The debugging data contract has a hard coded dependency on this value of MethodTable::Flags. If you change this value you must bump major_version_number.");
+    // TODO, andrewau, bump version number, testing
```

Comment:
You need to bump the version number [here](https://github.com/dotnet/runtime/blob/8a5283fcf1526e80255a35acbb8f9bc362c9f050/src/coreclr/nativeaot/Runtime/DebugHeader.cpp#L80).